pages/02.Documentation/07.images/default.en.md

```diff
@@ -16,24 +16,27 @@ You can view and manage images both via the OpenStack API, as well as using the
 
 ## Provided Images
 
-Images that are marked with **sys11 optimized** are built, optimized, tested and supported by Sys11.  
-More images will be added in the future.
-
-Distro           | Name                             | Description                             | Supported |
------------------|----------------------------------|-----------------------------------------|-----------|
-Ubuntu 16.04 LTS | Ubuntu 16.04 LTS                 | Unmodified, directly from Canonical     | yes       |
-Ubuntu 16.04 LTS | Ubuntu 16.04 LTS sys11 optimized | Modified, optimized for SysEleven Stack | yes       |
-Ubuntu 14.04 LTS | Ubuntu 14.04 LTS                 | Unmodified, directly from Canonical     | yes       |
-Ubuntu 14.04 LTS | Ubuntu 14.04 LTS sys11 optimized | Modified, optimized for SysEleven Stack | yes       |
-                 |                                  |                                         |           |
-Ubuntu 16.04 LTS | Rescue Ubuntu 16.04 sys11        | Modified, optimized for SysEleven Stack | yes       |
-
+Images with the description **sys11 optimized** are built, optimized, tested and fully supported by SysEleven.  
+Standard cloud images from their original source are supported with best effort only.
+
+Distro           | Name                             | Description                               | Supported   |
+-----------------|----------------------------------|-------------------------------------------|-------------|
+Ubuntu 18.04 LTS | Ubuntu 18.04 LTS                 | Unmodified, directly from Canonical       | best-effort |
+Ubuntu 18.04 LTS | Ubuntu 18.04 LTS sys11 optimized | Modified, optimized for SysEleven Stack   | yes         |
```

Comment:
I don't see a "Ubuntu 18.04 LTS sys11 optimized" image in my projects

Reply:
Sorry - forgot to add WIP on friday. Bionic images dev images work fine. Building production ready images today.